exchange/exchange.go

```diff
@@ -139,8 +139,8 @@ func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidReque
 	conversions := e.currencyConverter.Rates()
 
 	adapterBids, adapterExtra := e.getAllBids(auctionCtx, cleanRequests, aliases, bidAdjustmentFactors, blabels, conversions)
-	auc := newAuction(adapterBids, len(bidRequest.Imp))
 	bidCategory, err := applyCategoryMapping(requestExt, &adapterBids, *categoriesFetcher, targData)
```

Comment:
You may need `applyCategoryMapping()` to return the new `adapterBids`, as I recall it is a slice, which will do a lot with pointers.